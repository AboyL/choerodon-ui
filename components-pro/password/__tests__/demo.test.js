import demoTest from '../../../tests/shared/demoTest';

demoTest('password-pro');
