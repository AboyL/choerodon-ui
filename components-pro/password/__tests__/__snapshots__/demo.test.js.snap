// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components-pro/password/demo/basic.md correctly 1`] = `
<div
  class="c7n-row"
  style="margin-left:-5px;margin-right:-5px"
>
  <div
    class="c7n-col-12"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-password-wrapper c7n-pro-password-empty"
    >
      <input
        class="c7n-pro-password-fix-autofill"
        tabindex="-1"
      />
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-password"
          placeholder="请输入密码"
          type="password"
          value=""
        />
        <div
          class="c7n-pro-password-inner-button"
        >
          <i
            class="icon icon-visibility_off"
          />
        </div>
      </label>
    </span>
    <span />
  </div>
  <div
    class="c7n-col-12"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-password-wrapper c7n-pro-password-empty"
    >
      <input
        class="c7n-pro-password-fix-autofill"
        tabindex="-1"
      />
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-password"
          placeholder="无揭示按钮"
          type="password"
          value=""
        />
      </label>
    </span>
    <span />
  </div>
</div>
`;

exports[`renders ./components-pro/password/demo/controlled.md correctly 1`] = `
Array [
  <span
    class="c7n-pro-password-wrapper"
  >
    <input
      class="c7n-pro-password-fix-autofill"
      tabindex="-1"
    />
    <label>
      <input
        autocomplete="off"
        class="c7n-pro-password"
        type="password"
        value="default"
      />
      <div
        class="c7n-pro-password-inner-button"
      >
        <i
          class="icon icon-visibility_off"
        />
      </div>
    </label>
  </span>,
  <span />,
]
`;

exports[`renders ./components-pro/password/demo/dataset.md correctly 1`] = `
Array [
  <span
    class="c7n-pro-password-wrapper c7n-pro-password-required"
  >
    <input
      class="c7n-pro-password-fix-autofill"
      tabindex="-1"
    />
    <label>
      <input
        autocomplete="off"
        class="c7n-pro-password"
        name="first-name"
        type="password"
        value="choerodon-pro"
      />
      <div
        class="c7n-pro-password-inner-button"
      >
        <i
          class="icon icon-visibility_off"
        />
      </div>
    </label>
  </span>,
  <span />,
]
`;
